### PR TITLE
Feature/tb action 21

### PR DESCRIPTION
**Pull request by Kazalor**
Added running.py and imported it into main.py which creates a server on repl.it which can then be connected to the hosting service.

### CODE REVIEW

File: running.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Reference: https://replit.com/talk/learn/my-bot-goes-down-every-minute-and-doesn/11008/150912
```
Comment (EnderStir) [inline running.py:5]: Please add a comment at the top of this file explaing why it was added.

File: main.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Reference: https://realpython.com/how-to-make-a-discord-bot-python/
```
Comment (EnderStir) [inline main.py:1]: These changes are fine.

**Review by EnderStir — CHANGES_REQUESTED** [review]
Just a little more information in files is needed.

File: running.py (hunk 1 above)
Comment (Kazalor) [reply running.py:5]: The explanation has been added

**Review by EnderStir — APPROVED** [review]
Ready to be merged.

**Pull request merged**